internal/cli/iam/users/invite.go

```diff
@@ -63,14 +63,14 @@ func (opts *InviteOpts) newUserRequest() (*store.UserRequest, error) {
 	}
 	user := &store.UserRequest{
 		AtlasRoles: atlasRoles,
-		Country:    opts.country,
 		User: &opsmngr.User{
 			Username:     opts.username,
 			Password:     opts.password,
 			FirstName:    opts.firstName,
 			LastName:     opts.lastName,
 			EmailAddress: opts.email,
 			MobileNumber: opts.mobile,
+			Country:      opts.country,
```

Comment:
not about his line but I can't flag that one, 241, we mark country as required for cloud, could we do the same for cloud manager?